R/TimePointsToDate.R

```diff
@@ -0,0 +1,36 @@
+#'Numerically provided time points are converted to a POSIXct object in the
+#'format "%Y-%m-%d %H:%M:%OS".
+#'
+#'@param data : dataset with time points in it
+#'@param t : character time points field in the given dataset
+#'@param origin : character indicating the start of the time points
+#'(default: \code{origin = "1970-01-01"})
```

Comment:
The POSIXct object has to have an origin and if we want to consider hours/minutes/seconds we should use `POSIXct ` rather than `Date`. I couldn't find datasets with dates in mia or in curatedMetagenomiceData but GrienesenTSData has it. I can get the origin from there, but it is already given as `Date` object, so it doesn't help to turn numeric value to `Date` as we wanted.